extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -144,10 +144,17 @@ export class Bind {
     this.overridableKeys_ = [];
 
     /**
-     * Upper limit on number of bindings for performance.
+     * Upper limit on total number of bindings.
+     *
+     * The initial value is set to 1000 which, based on ~2ms expression parse
+     * time, caps "time to interactive" at ~2s after page load.
+     *
+     * User interactions can add new bindings (e.g. infinite scroll), so this
+     * can increase over time to a final limit of 2000 bindings.
+     *
      * @private {number}
      */
-    this.maxNumberOfBindings_ = 1000; // Based on ~2ms to parse an expression.
+    this.maxNumberOfBindings_ = 1000;
```

Comment:
Nit: The value 1000 is the DEFAULT_ALLOWED_BINDING_COUNT, and then references to the value are a little easier to follow below when doing a...

`min(MAX_ALLOWED_BINDING_COUNT, max(DEFAULT_ALLOWED_BINDING_COUNT, current + 500))`

Reply:
Good idea, will address in a follow-up.